8_Clock.py

```diff
@@ -0,0 +1,56 @@
+import datetime
+import os
+import time
+from colorama import init
+init()
+from colorama import Fore
+
+
+def read_number_shablon():
+    with open("numbers", "r", encoding="utf-8") as file_numbers:
+        text_number = file_numbers.readlines()
+    text_number_new = []
+    for i in text_number:
+        i_new = i.replace("\n", "")
+        text_number_new.append(i_new)
+    return text_number_new
+
+while True:
+    os.system('cls')
+    
+
+    current_date_time = datetime.datetime.now()
+    current_time = current_date_time.time()
+    out_time = current_time.strftime("%H:%M:%S")
```

Comment:
":" are redundant here...
You have to deal with it below
And I think you don't need to extract seconds

Reply:
Seconds are needed later for the color to change and the dots to flash